README.md

```diff
@@ -65,6 +65,9 @@ Enables you to create more customized BLE Peripheral's compared to the basic UAR
    * LC
  * [MPIDE](http://chipkit.net/started/install-chipkit-software/) ([ChipKit](http://chipkit.net))
    * PIC32MX
+ * [Arduino](https://www.arduino.cc/en/Main/arduinoBoardMega) ([Arduino Mega](https://www.arduino.cc/en/Main/arduinoBoardMega))
+   * Arduino MEGA
+   * You may experience memory problems on Arduino boards with 32kb flash storage (e.g. Uno,Leonardo)
```

Comment:
@ayildirim I've made the following updates to the read me: 
 * https://github.com/sandeepmistry/arduino-BLEPeripheral/commit/1b46c8847766a889a8f308e0858ca441a0d36a5f
 * https://github.com/sandeepmistry/arduino-BLEPeripheral/commit/60de4cfb77d5708ab5dc63de1f370bd0bcf5cd0d